packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -388,7 +704,7 @@ const DialogueCheckBox: React.FC<{
         }
       }
     `)
-
+  
   async function handleNewMatchAnonymisedAnalytics() {
     captureEvent("newDialogueReciprocityMatch", {}) // we only capture match metadata and don't pass anything else
```

Comment:
second argument is unnecessary

Reply:
Leaving to specifically highlight no data is being sent